articles/getting-started/setting-up-domains.md

```diff
@@ -16,12 +16,14 @@ The following are instructions for creating sending domains via the SparkPost UI
 
 1. Depending on what domain provider you are using, their DNS may look different, but how it works is essentially the same across all providers. There will be a TEXT field, and you can put the DKIM record in the text fields. We have provided you with many domain provider’s steps to go through to put your DKIM record into their records [here](https://www.sparkpost.com/docs/getting-started/getting-started-sparkpost/#prerequisites).
 
-1. Paste the "value" portion DKIM into your DNS TXT record for the hostname provided in the UI. Please note that the hostname for this record will be "domainkey._yoursendingdomain", and not the domain itself.
+1. To add the DKIM record to your DNS, first copy the "Hostname" portion provided in the _Set Up For Sending_ section of the SparkPost UI and paste it in the "Host" section of your DNS TXT record. This value should be `scphxxxx._domainkey.yourdomain.xxx`. Please note that some providers are particular about the "Host" and may require that the value be added as just `scphxxxx._domainkey`.
```

Comment:
Each DKIM domain name is different so let's make that clear. How about: `This value should be similar to scph...`